rules/linux/file_event/file_event_lnx_susp_shell_script_under_profile_directory.yml

```
@@ -0,0 +1,26 @@
+title: Potential Suspicious Shell Script Creation in Profile Folder
+id: 13f08f54-e705-4498-91fd-cce9d9cee9f1
+status: experimental
+description: Detects the creation of shell scripts under the "profile.d" path.
+references:
+    - https://blogs.jpcert.or.jp/en/2023/05/gobrat.html
+    - https://jstnk9.github.io/jstnk9/research/GobRAT-Malware/
+    - https://www.virustotal.com/gui/file/60bcd645450e4c846238cf0e7226dc40c84c96eba99f6b2cffcd0ab4a391c8b3/detection
+    - https://www.virustotal.com/gui/file/3e44c807a25a56f4068b5b8186eee5002eed6f26d665a8b791c472ad154585d1/detection
+author: Joseliyo Sanchez, @Joseliyo_Jstnk
+date: 2023/06/02
+tags:
+    - attack.persistence
+logsource:
+    product: linux
+    category: file_event
+detection:
+    selection:
+        TargetFilename|contains: '/etc/profile.d/'
+        TargetFilename|endswith:
+            - '.csh'
+            - '.sh'
+    condition: selection
+falsepositives:
+    - Legitimate shell scripts in the "profile.d" directory could be common in your environement. Apply additional filter accordingly via "image", by adding specific filenames you "trust" or by correlating it with other events.
+level: medium
```

```suggestion
level: low
```
I even think this is more of a low rule as-is. You would need to exclude typical package managers but I am not sure if that is even enough.

---

added another false positive test: "Regular file creation during system update or software installation by the package manager"